SimTracker/SiPixelDigitizer/plugins/SiPixelDigitizerAlgorithm.cc

```diff
@@ -1588,6 +1644,33 @@ void SiPixelDigitizerAlgorithm::pixel_inefficiency(const PixelEfficiencies& eff,
   std::vector<double> pixelEfficiencyROCStdPixels(16,1);
   std::vector<double> pixelEfficiencyROCBigPixels(16,1);
   
+  std::unique_ptr<PixelIndices> pIndexConverter(new PixelIndices(numColumns,numRows));  
+  
+  std::vector<int> badRocsFromFEDChannels(16,0);  
+  if (eff.PixelFEDChannelCollection_ != nullptr){
+    PixelFEDChannelCollection::const_iterator it = eff.PixelFEDChannelCollection_->find(detID);
+    if (it != eff.PixelFEDChannelCollection_->end()){      
+      for(const auto& ch: *it) {	
+	for (unsigned int i_roc = ch.roc_first; i_roc <= ch.roc_last; ++i_roc){
```

Comment:
Indentation is off.

Reply:
fixed

Reply:
In GitHub the indentation still looks off. This appears to be caused by mixture of tabs and spaces for the indentation (using only tabs or spaces would be better).